### ISSUE TITLE
New vector docs Leftnav accessibility: in light mode, some headings fail color contrast? 

### ISSUE THREAD

**Issue by johnsmartco** [riptano/docs-ui#93]
Hey guys, let me know what you find. 

For our Leftnav, in Light node, If i'm checking the correct set of foreground text color & background color hex # values, I think certain headings fail WCAG accessibility requirements.

https://webaim.org/resources/contrastchecker/?fcolor=8E9093&bcolor=F7F7F7

Do I have the right values?

Please refer to the leftnav when viewed from a page like https://docs.datastax.com/en/astra/astra-db-vector/api-reference/data-api-commands.html#advanced-feature-indexing-clause-on-createcollection , and the attached screenshot.  I tried to inspect the css colors but not sure if I've got the right values in that webaim comparison.

If we fail accessibility color contrast, it could be difficult for people who have low vision or who are color blind. 

<img width="324" alt="Screenshot 2024-02-01 at 11 34 18 AM" src="https://github.com/riptano/docs-ui/assets/4041525/a040ac5e-329b-4e44-884b-8ad072adbe49">

**Comment by johnsmartco** [comment]
cc: @colegoldsmith , @eric-schneider

**Comment by johnsmartco** [comment]
@colegoldsmith looked and the actual foreground/background colors for level-2 leftnav entries are:

https://webaim.org/resources/contrastchecker/?fcolor=6B6F73&bcolor=F7F7F7

... which passes AA compliance.  Closing this issue.
